se/se_epub_build.py

```diff
@@ -762,10 +762,9 @@ def build(self, run_epubcheck: bool, check_only: bool, build_kobo: bool, build_k
 						css = file.read()
 						processed_css = css
 
-						# Instead of targeting all spans with poetry CSS, only target proper verse spans (not inserted koboSpans)
-						processed_css = regex.sub(r"""(\.epub-type-z3998-(poem|verse|song|hymn)(.*))span(?![[.])""", r"""\1span.kobo-verse-span""", processed_css)
-						# Indentation selectors need to be more specific as well to compensate
-						processed_css = regex.sub(r"""p span\.i\d+""", r"""\g<0>.kobo-verse-span""", processed_css)
+						# Retarget span selectors at se spans (i.e. not koboSpans) only.
+						# Ignore span followed by a colon (this would be a CSS property like column-span).
+						processed_css = regex.sub(r"""(?<=\s)span(?=[^\w:])""", r"""span.se""", processed_css)
```

Comment:
We also have to test for the case where `span` is at start of the line, like

```css
span{
}
```

Unlikely to actually occur but we don't want to surprised later down the line if it actually does.

Reply:
So this regex already matches that case:

```
span.se{
}
```

`\s` includes newline, so it sees that the span is preceded by a newline and converts it into `span.se`. Theoretically this wouldn't work if the pattern you described was at the very start of the file, but that isn't possible in our case since the `@charset` and @namespace epub` are always at the top of every SE CSS file.